README.md

```diff
@@ -6,56 +6,27 @@ We use a template repository approach for maintainability.
 
 ## Creating a repository for an individual training workshop
 
-First, you will need to create a new repository.
+First, you will need to create a new repository using the [`Use the template` button](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-from-a-template).
+This will create a new repository with the same layout as this template repository under the given organization or user you choose to administer the repository; if you are a Data Lab member running an internal training workshop, the `AlexsLemonade` organization should administer the new repository.
 
-### Data Lab workshops
+We recommend setting the new repository to public.
+(When you use GitHub pages on a private repository, those pages are publicly accessible anyway.)
 
-This repository can be used as a template by anyone with read permissions.
-Select the [`Use the template` button](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-from-a-template) and create the repository under the organization (typically `AlexsLemonade`) or user you would like to administer the repository.
-We recommend setting the new repository to public. (When you use GitHub pages on a private repository, those pages are publicly accessible anyway.)
+## Customizing the new repository for an individual training workshop
 
-### Workshops hosted by others
+We have set up a [manually-triggered GitHub Action](https://docs.github.com/en/actions/using-workflows/manually-running-a-workflow) to support setting up this new repository.
+When triggered, this action will file a series of issues comprising tasks that should be completed before the start of a workshop.
+You should trigger this action once the [`training-modules` repository](https://github.com/AlexsLemonade/training-modules) tag for the given workshop has been created, as one of the _first steps_ of setting up your new repository.
 
-If you are an external user, you can either [file an issue to request read access](https://github.com/AlexsLemonade/training-specific-template/issues/new?assignees=&labels=request&template=request-read-access-to-use-template-for-a-workshop.md&title=%5BRequest+read+access%5D+) or fork the repository.
-If you request read access and it is granted, follow [the instructions above](#data-lab-workshops).
-To fork the repository, [follow the GitHub instructions for forking a repository](https://help.github.com/en/github/getting-started-with-github/fork-a-repo).
+To run this action, navigate to the "Actions" tab.
+On the left, you will see all available workflows.
+Click the workflow named `Manually trigger issue creation for standard set up`, and then click the "Run workflow" dropdown button.
+You will need to provide one required input, the `training-modules` repository release tag you would like to associate with the workshop.
```

Comment:
Can this be updated later? If so, we should say that here, and probably mention the default. Or if not, we might say whether this release tag needs to exist, or if it is the one we _will_ assign?

Reply:
Hm, looks like the _only_ time this tag is used is for the title of an issue!

https://github.com/AlexsLemonade/training-specific-template/blob/615e672511c8abf0e2ef92d62f98649851c5e2dd/.github/workflows/file-setup-issues.yml#L80-L86

I think it's worth just getting rid of this input since there's enough info in the issue template itself to create the release:
https://github.com/AlexsLemonade/training-specific-template/blob/main/setup-issue-templates/workshop-release.md